src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+// NOTE GATE
+  return (!a);
 };
 
 function both(a, b) {
-  // your code here
+// AND GATE
+  return (a&&b);
 };
 
 function either(a, b) {
-  // your code here
+// OR GATE
+  return (a||b);
 };
 
 function none(a, b) {
-  // your code here
+// NOR GATE
+  return !(a||b);
 };
 
 function one(a, b) {
-  // your code here
+// XOR GATE
+  return (a && !b) || (!a && b);
 };
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a===b;
```

Comment:
Very nice use of the === strict equality operator, well done :)

For better formatting you should keep spaces between the variables and the operator:
`````
a === b
``````